.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
```

Comment:
I don't really like the idea of using "request" word here. "Request" carries meaning of getting response, and "request for something" typically implies approval or denial of that request.

Actual meaning of issue like that is rather dual: it could be either declaration that a person wants it implemented (but is not willing to do it), or it could be declaration of the work being started (and thus person will actively participate in implementation and invites other to join).

Neither of these qualifies for "request", in my opinion. Something neutral, and one-size-fits-all-these-cases, like `new-spec` will better, because:

* It won't imply that some higher force (i.e. repo owners or whoever) needs to "approve" or "reject" this request
* It won't imply that person requesting format will automatically get someone (i.e. repo owners) working on it

Reply:
Hey, sorry to butt into the issue, I just want to give my two cents.

>It won't imply that some higher force (i.e. repo owners or whoever) needs to "approve" or "reject" this request

While `request` can imply some form of approval or rejection, I would argue that any other form of requesting or suggesting something would imply a form of approval or rejection. Are we asking for a format? That could be rejected. Are we wanting a format? You could say "We won't do that." Even replacing `request` for `suggestion for` here may not make a difference -- suggestions can just as easily be rejected as requests can.

> It won't imply that person requesting format will automatically get someone (i.e. repo owners) working on it

I would argue that `request` does not imply this anyways. `Request`, according to Google, is `an act of asking politely or formally for something.` As such, I would believe that a request is the person asking for something to be fulfilled -- whether or not it is *actually* fulfilled is another matter entirely, and thus whether this we label this as a request or as a suggestion (or as some other word) wouldn't be of relevance anyways.

I don't intend to deny that there could be some better word we could use here, one that might better fit what that issue template is meant for, but I don't know if there's any point to being concerned, especially if it's only about the implications of a word. At least, not until we know for sure whether people will actually take something in a certain way or not.

Reply:
> Are we asking for a format? That could be rejected. Are we wanting a format? You could say "We won't do that."

My very point is that this type of issue can't be actually "accepted" or "rejected".

If somebody is asking for a format — we, as centralized repository, are not really in position to say "this is absolutely out of limits". Unless the issue is not totally uncomprehensible/irrelevant — like some kind of spam (and thus had to be closed as "invalid"), it is welcome to stay. May be some day somebody will come by and do something about it.

Reply:
>My very point is that this type of issue can't be actually "accepted" or "rejected".

Then why are we concerned about the potential implication of needing an acception or rejection?

Reply:
Sorry, I don't quite follow the logic of the argument. We aren't concerned, so what? We aren't concerned about zillions of other things here as well, so we should mention all of them?

Reply:
> Sorry, I don't quite follow the logic of the argument.

To my understanding, a request could go one of three ways: it could be accepted or rejected, or it could be left there until someone decides to pick it up and act on it. A suggestion or a want for something could easily go the one of those same three ways. You said `"request for something" typically implies approval or denial of that request`, and I want to dispute that based on my understanding of what a request is. So I'm saying that I don't think that a request necessarily implies needing approval/rejection, especially as any other word that you could use in that context would end up meaning the same thing anyways.